musicbot/bot.py

```diff
@@ -2028,6 +2031,8 @@ def check(reaction, user):
 
         num_voice = sum(1 for m in voice_channel.members if not (
             m.voice.deaf or m.voice.self_deaf or m == self.user))
+        if num_voice == 0:
+            num_voice += 1 # add one incase all users are deafened, to avoid divison by zero
```

Comment:
For the purpose of what?

Reply:
If all of the people that are on the channel are deafened, it would throw a ZeroDivisionError, adding 1 makes it so it would allow skipping even when everyone is deafened.